src/clarity-angular/layout/tabs/aria-service.ts

```diff
@@ -0,0 +1,7 @@
+import {Injectable} from "@angular/core";
+
+@Injectable()
+export class AriaService {
```

Comment:
I considered this too, but different components need different attributes. I looked at it quickly, and it seemed to me like there was very little reusability. 😞